Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "chronicle-inx"
 version = "0.1.0"
-authors = [ "IOTA Stiftung" ]
+authors = ["IOTA Stiftung"]
 edition = "2021"
-description = "IOTA permanode implemented as an IOTA Node Extension (INX)"
+description = "IOTA permanode implemented as an IOTA Node Extension (INX)."
 readme = "README.md"
 repository = "https://github.com/iotaledger/inx-chronicle"
 license = "Apache-2.0"
-keywords = [ "iota", "storage", "permanode", "node", "inx" ]
+keywords = ["iota", "storage", "permanode", "node", "inx"]
```

Comment:
"permanode" and "node" are a bit redundant. I'ld replace "node" for "chronicle" as this is the brand name.